fm2prof/RegionPolygonFile.py

```diff
@@ -124,7 +126,9 @@ def classify_points_with_property_rtree_by_polygons(
         del idx
         return np.array(point_properties_list)
 
-    def __get_polygon_property(self, grouped_values: list, property_name: str) -> str:
+    def __get_polygon_property(
+        self, grouped_values: list, property_name: str
+    ) -> str:  # TODO: Can this be removed?

```

Comment:
And, can it be removed? :D Or are you not sure yet?

Reply:
I'm not sure, the method is not used anywhere and it is an internal method. I'll keep it in. Koen Berends can always remove it later when he is back.